rmgpy/molecule/resonance.py

```diff
@@ -130,34 +130,107 @@ def analyzeMolecule(mol):
 
 def generateResonanceIsomers(mol):
     """
-    Generate and return all of the resonance isomers of this molecule.
+    Generate and return all of the resonance structures for the input molecule.
```

Comment:
I find this summary in a previous PR is helpful, please somehow add it to the docstring?:

### Other Resonance Structures
For aromatic species, the process is now to first generate the aromatic resonance structure(s), which gives a consistent starting point regardless of the input. Then, the process diverges based on the features of the molecule.
- For radical polycyclic aromatic species, Kekule structures are generated in order to generate adjacent resonance structures. The resulting structures are then used for Clar structure generation. After all three steps, any non-aromatic structures are removed, under the assumption that they are not important resonance contributors.
- For radical monocyclic aromatic species, Kekule structures are generated along with adjacent resonance structures. All are kept regardless of aromaticity because the radical is more likely to delocalize into the ring. Ideally, the aromatic structures would be weighted more than the non-aromatic ones, but that is for future development.
- For stable polycyclic aromatic species, only Clar structures are generated
- For stable monocyclic aromatic species, Kekule structures are generated